examples/slack-security-findings/daily_findings.py

```diff
@@ -0,0 +1,188 @@
+from __future__ import annotations
+
+import json
+import os
+import sys
+from datetime import date, timedelta
+from http.client import RemoteDisconnected
+from json import JSONDecodeError
+from string import Template
+from typing import TypedDict, Callable, Any
+from urllib import request
+from urllib.error import URLError
+import logging
+from argparse import ArgumentParser
+
+LOG = logging.getLogger(__name__)
+
+
+class Finding(TypedDict):
```

Comment:
`TypedDict` is a bit weird, you can happily give it fields it doesn't know, and it will happily allow it. `dataclass` is a bit more type-safe, and you can still use `asdict` to convert that into a dict. This is more pedantic, not a huge gain one way or the other.

Reply:
Switched over to `@dataclass`, which is nicer anyway!